.clj-kondo/config.edn

```diff
@@ -0,0 +1 @@
+{:config-paths ["adambard/failjure"]}
```

Comment:
@pratik97 This should be `../resources/clj-kondo.exports/adambard/failjure`

Reply:
@borkdude I think @ericdallo mentioned on Slack that `resources` folder is already on the classpath(lein adds that), and we don't need to mention the relative path here. It's similar as one of Eric's [PR](https://github.com/ptaoussanis/encore/pull/56/files).

Reply:
The above only works if you invoke clj-kondo with `--copy-configs` so it will copy `resources/clj-kondo.exports/adambard/failjure` to `.clj-kondo/adambard/failjure` which feels a bit unnecessary since the files are already locally available and makes it more difficult to develop the config. Having only one source of truth is better. `:config-paths` doesn't work with a classpath: it just looks up a directory relative to the `config.edn` file. So `../resources/clj-kondo.exports/adambard/failjure` is what is recommended to use here.
See fulcro's config for comparison:
https://github.com/fulcrologic/fulcro/blob/develop/.clj-kondo/config.edn

Reply:
Updated the path

Reply:
@borkdude does this look fine? If so, @adambard can we merge this?